app/code/Magento/MediaGalleryUi/Ui/Component/Listing/Columns/Url.php

```diff
@@ -98,7 +98,7 @@ public function prepare(): void
             array_replace_recursive(
                 (array)$this->getData('config'),
                 [
-                    'onInsertUrl' => $this->urlInterface->getUrl('cms/wysiwyg_images/oninsert'),
+                    'onInsertUrl' => $this->urlInterface->getUrl('media_gallery_renditions/image/oninsert'),
```

Comment:
The issue with this approach is that it makes MediaGalleryUi module dependent on Renditions. It's better to have the reversed dependency.

Let's keep the plugin in Renditions module and move the introduced controller to MediaGalleryUi for this purpose